data-prepper-plugins/date-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/date/DateProcessorConfig.java

```diff
@@ -20,6 +21,16 @@
 import java.util.Locale;
 import java.time.format.DateTimeFormatter;
 
+@ConditionalRequired(value = {
+        @ConditionalRequired.IfThenElse(
+                ifFulfilled = {@ConditionalRequired.SchemaProperty(field = "match", value = "null")},
```

Comment:
Nit: Having a static import for `@ConditionalRequired.SchemaProperty` probably will make this statement more easy to read